subsys/usb/usb_c/usbc_pe_common.h

```diff
@@ -218,4 +320,120 @@ enum tc_power_role pe_get_power_role(const struct device *dev);
  */
 enum tc_cable_plug pe_get_cable_plug(const struct device *dev);
 
+/**
+ * @brief Sets a Policy Engine state
+ */
+void pe_set_state(const struct device *dev,
+		  const enum usbc_pe_state state);
+
+/**
+ * @brief Get the Policy Engine's current state
+ */
+enum usbc_pe_state pe_get_state(const struct device *dev);
+
+/**
+ * @brief Get the Policy Engine's previous state
+ */
+enum usbc_pe_state pe_get_last_state(const struct device *dev);
+
+/**
+ * @brief Send a soft reset message
+ */
+void pe_send_soft_reset(const struct device *dev,
+			const enum pd_packet_type type);
+
+/**
+ * @brief Send a Power Delivery Data Message
+ */
+void send_data_msg(const struct device *dev,
+		   const enum pd_packet_type type,
+		   const enum pd_data_msg_type msg);
+
+/**
+ * @brief Send a Power Delivery Control Message
+ */
+void send_ctrl_msg(const struct device *dev,
+		   const enum pd_packet_type type,
+		   const enum pd_ctrl_msg_type msg);
+
+/**
+ * @brief Request desired voltage from source.
+ */
+void pe_send_request_msg(const struct device *dev,
+			 const uint32_t rdo);
+
+/**
+ * @brief Transitions state after receiving an extended message.
+ */
+void extended_message_not_supported(const struct device *dev);
+
+/**
+ * @brief Handle common DPM requests
+ *
+ * @retval True if the request was handled, else False
+ */
+bool common_dpm_requests(const struct device *dev);
+
+/**
+ * @brief Handle sink-specific DPM requests
+ */
+bool sink_dpm_requests(const struct device *dev);
```

Comment:
Should this be in the sink header file?

Reply:
This was changed in a later PR but should be done here. I've removed it.